mp/src/materialsystem/stdshaders/pbr_dx9.cpp

```diff
@@ -539,6 +607,15 @@ BEGIN_VS_SHADER(PBR, "PBR shader")
 				pShaderAPI->SetPixelShaderConstant(PSREG_ENVMAP_TINT__SHADOW_TWEAKS, tweaks, 1);
 			}
 
+#ifdef		PBRPARALLAX
+			float flParams[4] = { 0.0f, 0.0f, 0.0f, 0.0f };
+			flParams[0] = 50;
```

Comment:
What are these constants?

Reply:
initialization.

Reply:
Can you document what the values mean? You're currently initializing 50, 50 which are unitless.

Reply:
This is no longer being used right?